### PR TITLE
Textarea mode variant 3118

### PR DESCRIPTION
**Pull request by EmelieLitwin**
**Describe pull-request**  
Changed mode variant class and added variables for dark mode

**Solving issue**  
Fixes: [AB#3119](https://dev.azure.com/scaniadigitaldesignsystem/3d9eb9c6-0045-473d-bc1d-e842dd779200/_workitems/edit/3119)

### CODE REVIEW

**Review by nathalielindqvist — APPROVED** [review]
LGTM! 👍

File: tegel/src/components/textarea/textarea.tsx [hunk 1]
```diff
@@ -133,33 +133,32 @@ export class Textarea {
...
-            />
-            <path
-              fill-rule="evenodd"
-              clip-rule="evenodd"
-              d="M1.43306 1.43306C1.67714 1.18898 2.07287 1.18898 2.31695 1.43306L18.5671 17.6832C18.8112 17.9273 18.8112 18.323 18.5671 18.5671C18.323 18.8112 17.9273 18.8112 17.6832 18.5671L1.43306 2.31695C1.18898 2.07287 1.18898 1.67714 1.43306 1.43306Z"
-              fill="currentColor"
-            />
-          </svg>
-</span>
+          <span class="sdds-textarea-icon__readonly">
```
Comment (mJarsater) [inline tegel/src/components/textarea/textarea.tsx:136]: Do we have our own sdds-icon for this?

**Review by mJarsater — CHANGES_REQUESTED** [review]

**Review by mJarsater — APPROVED** [review]
LGTM! Not really a part of this PR, but should the error state include a error icon before the error text below, like it does in textfield?

**Comment by EmelieLitwin** [comment]
> LGTM! Not really a part of this PR, but should the error state include a error icon before the error text below, like it does in textfield?

Yes it should 🤔 Excellent spotted. I'll create a separate ticket for it for better traceability

**Pull request merged**